src/index.ts

```diff
@@ -345,6 +349,54 @@ class Spanner extends GrpcService {
       [CLOUD_RESOURCE_HEADER]: this.projectFormattedName_,
     };
     this.directedReadOptions = directedReadOptions;
+    this.emulatorHost = emulatorHost;
+  }
+
+  // Helper for instance admin client API calls
+  get_instance_admin_client(this): void {
+    const clientName = 'InstanceAdminClient';
```

Comment:
The `prepareGapicRequest_(..)` function just does the following:

```
      if (!this.clients_.has(clientName)) {
        this.clients_.set(clientName, new v1[clientName](this.options));
      }
      const gaxClient = this.clients_.get(clientName)!;
```

I think that we should do exactly the same here, as they should both return the exact same client. That is:
1. If I for example call `getInstances(..)` on this class, then it will call `prepareGapicRequest_` with client name `InstanceAdminClient`. That will call the above code snippet. Any further calls to this `get_instance_admin_client` will then return the instance that was returned by the first call to `getInstances(..)`.
2. The same is also true the other way around: If I first call `get_instance_admin_client`, then the code in this method will be executed to create an `InstanceAdminClient`. Any subsequent call to `getInstances(..)` will then use the client created here.

The above therefore means that they are clearly intended to be the same, so they should also be created using the same code to prevent any differences between them.

That also means that the additional fields for `_instance_admin_api` and `_database_admin_api` are not needed.